news/2022-07-26-project-loved-july-2022.md

```diff
@@ -0,0 +1,385 @@
+
+[![](/wiki/shared/news/2022-07-26-project-loved-july-2022/1070119.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[403 - Northern Lights -RYUWAVE'S SPLITTERCORE BOOTLEG-](https://osu.ppy.sh/beatmapsets/1070119#taiko) by [\_yu68](https://osu.ppy.sh/users/6170507)\
+240 BPM, 2:10 | 8.64★, 9.21★\
+*description written by [Boaz](https://osu.ppy.sh/users/13302996)*
+
+\_yu68 returns to the Loved battlefield with yet another map for the real speed demons to enjoy. For those who don't know, "splittercore" is subgenre of speedcore that makes use of extremely high tempos, and this song is not any different. The Challenger difficulty features lots of 360 and 480 BPM bursts within 240 BPM streams, so the name isn't joking about this being a real challenge, even for the speediest among us.
+
+To the people who still wanted to play this without HT, it's your lucky day, since this mapset features a nerfed difficulty as well! Don't let it fool you though, it still includes the long 240 BPM streams with some 360 BPM bursts now and then.
+
+This concludes this month's osu!taiko Loved picks, we hope you all enjoyed them and make sure you don't get any injuries playing this map in particular!
+
+## osu!catch
+
+The osu!catch Loved candidates were chosen by [Kasumi-sama](https://osu.ppy.sh/users/6177263), [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600) and [Wesley](https://osu.ppy.sh/users/2407265)!
+
+---
+
+[![](/wiki/shared/news/2022-07-26-project-loved-july-2022/902888.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Jerma - Rat's Birthday Mixtape](https://osu.ppy.sh/beatmapsets/902888#fruits) by [Kyptoric](https://osu.ppy.sh/users/1361268)\
+117 BPM, 1:47 | 7.79★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Are you looking for a map that has plenty of jumps? Well, look no further, *Jerma - Rat's Birthday Mixtape* by Kyptoric will provide you with all the jumps you'll ever need. After only playing it once or twice you'll be ratJAMMING throughout the whole song, so give it a go and jam away!
+
+[![](/wiki/shared/news/2022-07-26-project-loved-july-2022/733396.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ARM feat. Amane (COOL&CREATE) & FOXMANN - Yamanba Techno de Chanabekko](https://osu.ppy.sh/beatmapsets/733396#fruits) by [dazzulay](https://osu.ppy.sh/users/1948043)\
+140 BPM, 5:04 | 5.76★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+After almost exactly four years, dazzulay returns to Loved with another really cool map. Overall it is clean and plays nice, with a few unrankable patterns here and there which do not greatly impact the way it plays. It is probably not very difficult for experienced players, so trying it with mods may be encouraged. Have fun!
+
+[![](/wiki/shared/news/2022-07-26-project-loved-july-2022/394927.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
```

Comment:
A note: this background/image should be replaced before the news article is published as it didnt pass the Content Review vote. (Vote 654)